.github/workflows/piper.yml

```diff
@@ -0,0 +1,73 @@
+name: Piper Example
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  push:
+    paths:
+      - ".github/workflows/piper.yml"
+      - "wasmedge-piper/**"
+  pull_request:
+    paths:
+      - ".github/workflows/piper.yml"
+      - "wasmedge-piper/**"
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Install Dependencies for building WasmEdge
+        run: |
+          sudo apt-get update
+          sudo apt-get install ninja-build
+
+      - name: Install ONNX Runtime 1.14.1
+        run: |
+          curl -LO https://github.com/microsoft/onnxruntime/releases/download/v1.14.1/onnxruntime-linux-x64-1.14.1.tgz
+          tar zxf onnxruntime-linux-x64-1.14.1.tgz
+          sudo mv onnxruntime-linux-x64-1.14.1/include/* /usr/local/include/
+          sudo mv onnxruntime-linux-x64-1.14.1/lib/* /usr/local/lib/
+          rm -rf onnxruntime-linux-x64-1.14.1.tgz onnxruntime-linux-x64-1.14.1
+          ldconfig
```

Comment:
Since we are checking out the WasmEdge codebase, should we use the install script from the WasmEdge repo instead?

Reply:
ok https://github.com/second-state/WasmEdge-WASINN-examples/pull/145/commits/c6fd2ba2b0b529c21f9a066c911509142f182b29